modules/ardrivo/src/Entrypoint.cxx

```diff
@@ -80,7 +80,12 @@ void merge_boards(BoardData old_board, BoardData& new_board) noexcept {
         for(auto [from, to] : ranges::views::zip(old_board.*mvec, new_board.*mvec))
             std::tie(to.rx, to.tx) = std::forward_as_tuple(from.rx, from.tx);
     };
-
-    copy_proto_bufs(&BoardData::i2c_buses);
     copy_proto_bufs(&BoardData::uart_buses);
+
+    auto copy_proto_slaves = [&]<class T>(std::vector<T>(BoardData::* mvec)) {
+        for(auto [from, to] : ranges::views::zip(old_board.*mvec, new_board.*mvec))
+            to.slaves = std::move(from.slaves);
+    };
```

Comment:
Wouldn't [`ranges::move`](https://ericniebler.github.io/range-v3/group__group-algorithms.html#gaa55485146994b298d1392f1aa421fdfe) be better?

Reply:
NVM it's not due to possibly mismatching sizes